borer-akka-serializer/src/main/scala-2.12/org/virtuslab/akkaserializationhelper/RuntimeReflections.scala

```diff
@@ -1,4 +1,4 @@
-package org.virtuslab.akkasaferserializer
+package org.virtuslab.akkaserializationhelper
```

Comment:
Just thinking whether this should be called `org.virtuslab.ash` in package names 🤔

Reply:
I agree